static/src/stylesheets/module/_social.scss

```diff
@@ -212,3 +212,50 @@ category: Common
     visibility: visible;
     opacity: 1;
 }
+
+// Testing static social on mobile
+
+.social-fixed {
+    height: 40px;
+    position: fixed;
```

Comment:
What about older mobile browser than don't support position fixed? We should only render this template if they support.